repaygithub/ui-tools
# CACTUS-298 :: Update dependencies for repay-scripts and babel-preset

**Note:** I had to alter a test to get everything to pass in the latest version of Jest.  The test was verifying that "watch" mode works under `@repay/scripts`.  It looks like we weren't able to re-require a module in a test body after it was updated, so I just deleted the first require and waited to require the module until it was updated.  That still tests what we want anyway.

### How to Test

1. Run `yarn link` inside `modules/babel-preset` and `modules/repay-scripts`
2. Head on over to Cactus and run `yarn link @repay/babel-preset` followed by `yarn link @repay/scripts`.
3. Run `yarn cleanup && yarn build`.  This will help test the Babel upgrades as well as the major version bump to Rollup.  Make sure that everything succeeds.
4. Run `yarn w mock-ebpp cleanup && yarn w mock-ebpp build`.  (You can do this for other example apps too.) This will test the webpack build upgrades.  Just make sure everything runs successfully.
5. Run `yarn w mock-ebpp start`.  Bring up the localhost app in your browser (bonus points: do this in IE too).  Make sure everything looks good/works well.  This should test the webpack dev server upgrades and will test that Babel and Rollup worked appropriately for the Cactus Web bundle.
6. Make sure tests pass in the `ui-tools` repo.
